docs/imm-otel.md

```diff
@@ -0,0 +1,101 @@
+- Use the [advanced installation](https://docs.signalfx.com/en/latest/integrations/agent/advanced-install-options.html "Advanced Installation Options") option for bulk deployments and for configuring various monitors for your environment.
+
+**Step 2: Configure the SignalFx Smart Agent**
+
+The SignalFx Smart Agent is configured primarily by a YAML document located at `/etc/signalfx/agent.yaml`. The location of the configuration file can be specified by the `-config` flag to the agent binary (`signalfx-agent`).
+
+See [Agent Configuration](https://docs.signalfx.com/en/latest/integrations/agent/config-schema.html "Agent Configuration") for configuration options and an example configuration file. See [deployments](https://github.com/signalfx/signalfx-agent/tree/master/deployments "Deployments") for more information on the various ways to deploy the SignalFx Smart Agent.
+
+## Deploy an OpenTelemetry Collector for SignalFx
+
+After installing the SignalFx Smart Agent on each host, you can deploy an OpenTelemetry Collector in each datacenter/region/cluster where traced applications run. In general, an OpenTelemetry Collector should receive data from the SignalFx Smart Agent.
+
+The OpenTelemetry Collector uses pipelines to receive, process, and export trace data with components conveniently known as receivers, processors, and exporters. Set up pipelines with services. You can also add extensions that provide an OpenTelemetry Collector with additional functionality, such as diagnostics and health checks. The OpenTelemetry Collector has two versions: a [core version](https://github.com/open-telemetry/opentelemetry-collector "Core Version") and a [contributions version](https://github.com/open-telemetry/opentelemetry-collector-contrib "Contributions"). The core version provides receivers, processors, and exporters for general use. The contributions version provides receivers, processors, and exporters for specific vendors and use cases.
+
+SignalFx uses the components described in the following table to send data to an OpenTelemetry Collector and to receive data from an OpenTelemetry Collector:
+
+| **Component** | **Name**        | **Description**                                                                                       |  
+|---------------| :---------------:  |-------------------------------------------------------------------------------------------------------|
+| Receiver      | `signal-fx`     | Component that sets the endpoint for receiving metrics data with the SignalFx metric data format.     |      
+| Processor     | `memory_limiter`, `batch`, and `queued_retry` are the recommended processors.         | Component that pre-processes data before it is exported.                                              |      
+| Exporter      | `signal-fx`     | Component that forwards data to SignalFx with the metric data format.                                 |
```

Comment:
Again, not sure the customer cares -- they need configuration, but likely do not need the specifics of things. Also missing processors here